### ISSUE TITLE
[BUG] Unable to untag expenses

### ISSUE THREAD

**Issue by mr-lanholmes** [Ivy-Apps/ivy-wallet#3203]
### Please confirm the following

- [X] I checked [the current issues](https://github.com/Ivy-Apps/ivy-wallet/issues) for duplicate problems

### Describe the bug

Once selected, I am unable to see the difference between selected and unselected expenses, as well as unable to unselect.

### To Reproduce

1. 

### Expected behavior

unable to see the difference between selected and unselected expenses, as well as unable to unselect.

### App version

4.6.3

### Smartphone

S24 Ultra

**Comment by ivywallet** [comment]
Thank you @mr-lanholmes for raising Issue #3203! 🚀
What's next? Read our **[Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md) 📚**.

_Tagging @ILIYANGERMANOV for review & approval 👀_

**Comment by rishi2062** [comment]
Hi @mr-lanholmes , is it possible for you to provide screenshot or recording about this issue, i'm having confusion on how to select and unselect expenses

**Comment by rishi2062** [comment]
Thanks a lot for providing recording, i was struggling since yesterday. Anyways will try to fix it asap.

**Comment by rishi2062** [comment]
I'm on it

**Comment by ivywallet** [comment]
Thank you for your interest @rishi2062! 🎉
Issue #3203 is assigned to you. You can work on it! ✅

_If you don't want to work on it now, please un-assign yourself so other contributors can take it._

Also, make sure to read our [Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md).

**Comment by ivywallet** [comment]
Thank you for your interest @rishi2062! 🎉
Issue #3203 is assigned to you. You can work on it! ✅

_If you don't want to work on it now, please un-assign yourself so other contributors can take it._

Also, make sure to read our [Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md).

**Comment by rishi2062** [comment]
@mr-lanholmes can you again verify the version if it is same or is it play store one..
and i believe yours is s24 ultra , so you must be having android 14

In both Pixel 7A emulator and samsung m23 with Api level 34 it is working fine, did this issue got solved already @ILIYANGERMANOV 
or any idea, if issue is still the case?
https://github.com/Ivy-Apps/ivy-wallet/assets/72225636/946fb36c-c95a-46f0-b753-f2016e3f2ebe

**Comment by rishi2062** [comment]
I cross checked it, it is not working with play store version, thanks for clarifying @mr-lanholmes 😃,

**Comment by rishi2062** [comment]
@ILIYANGERMANOV In both release mode and debug mode this functionality is working fine, is the playstore version up-toDate or do i need to do some extra setup to check this issue for play store version,

**Comment by rishi2062** [comment]
I don't know if there is any internal update to the app, but if thats not the case,
sometimes due to over work in UI thread, some animations can not work as expected, if i give you the number, it has to be under 16ms per frame for smooth animation, or in recent library versions, i saw some recomposition issue, that could be also a potential reason, but @ILIYANGERMANOV only can clear out things.
and can you tell me did your notice any shuttering or frame drop or lag while doing these activities in app?

**Comment by ILIYANGERMANOV** [comment]
> @ILIYANGERMANOV @rishi2062 despite this is "solved", i notice the bug still persist for newly created tag.
> 
> On a side note, the tag doesnt get exported (in json format) when backing up the data.

Is this with the GitHub APK or Google Play?

**Comment by ILIYANGERMANOV** [comment]
> @ILIYANGERMANOV is the same version Google Play 4.6.3 (163)

That's expected. The fix is only in the latest GitHub APK and haven't been released, yet. For more info:
https://t.me/c/1647280565/1/27319

Tl;dr; we're not sure if we can release right now because many other things might be broken

**Comment by rishi2062** [comment]
What should we do with this issue? @ILIYANGERMANOV
